CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -36,15 +36,19 @@ public static class EnvUtil
         public const string MsalFileCacheEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_ENABLED";
         public const string MsalFileCacheLocationEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_LOCATION";
 
-        private static readonly string LocalAppDataLocation = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.Create), "MicrosoftCredentialProvider");
+        private static readonly string LocalAppDataLocation = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.Create);
 
-        public static string AdalTokenCacheLocation { get; } = Path.Combine(LocalAppDataLocation, "ADALTokenCache.dat");
+        public static string AdalTokenCacheLocation { get; } = Path.Combine(LocalAppDataLocation, "MicrosoftCredentialProvider", "ADALTokenCache.dat");
 
-        public static string DefaultMsalCacheLocation { get; } = Path.Combine(LocalAppDataLocation, "MSALTokenCache.dat");
+        // from https://github.com/GitCredentialManager/git-credential-manager/blob/df90676d1249759eef8cec57155c27e869503225/src/shared/Microsoft.Git.CredentialManager/Authentication/MicrosoftAuthentication.cs#L277
+            // The Visual Studio MSAL cache is located at "%LocalAppData%\.IdentityService\msal.cache" on Windows.
+            // We use the MSAL extension library to provide us consistent cache file access semantics (synchronisation, etc)
+            // as Visual Studio itself follows, as well as other Microsoft developer tools such as the Azure PowerShell CLI.
```

Comment:
I'd prefer the indentation be _in_ the comment:
```suggestion
        //     The Visual Studio MSAL cache is located at "%LocalAppData%\.IdentityService\msal.cache" on Windows.
        //     We use the MSAL extension library to provide us consistent cache file access semantics (synchronisation, etc)
        //     as Visual Studio itself follows, as well as other Microsoft developer tools such as the Azure PowerShell CLI.
```